pkg/registry/apps/correlations/register.go

```diff
@@ -0,0 +1,44 @@
+package correlations
+
+import (
+	restclient "k8s.io/client-go/rest"
+
+	"github.com/grafana/grafana-app-sdk/app"
+	appsdkapiserver "github.com/grafana/grafana-app-sdk/k8s/apiserver"
+	"github.com/grafana/grafana-app-sdk/simple"
+	"github.com/grafana/grafana/apps/correlations/pkg/apis"
+	correlationsapp "github.com/grafana/grafana/apps/correlations/pkg/app"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+var (
+	_ appsdkapiserver.AppInstaller = (*CorrelationsAppInstaller)(nil)
+)
+
+type CorrelationsAppInstaller struct {
+	appsdkapiserver.AppInstaller
+	cfg *setting.Cfg
```

Comment:
Lets see if we really need `setting.Cfg` -- this dependency brings in a ton of stuff that will be difficult to maintain in a MT environment, so lets only add it if it is really necessary

Reply:
Okay, but please update the how to guide with a template of what we should have instead. This is in step 13. Internal document only. https://docs.google.com/document/d/1UG3Ayno4aoiedG9YsITGDneRvkm6vMVYpdeNG6u5Hf8/edit?tab=t.0#heading=h.5qp6yczcp8h